network-config-analyzer/K8sPolicyYamlParser.py

```diff
@@ -200,7 +200,10 @@ def parse_ip_block(self, block):
         """
         self.check_fields_validity(block, 'ipBlock', {'cidr': [1, str], 'except': [0, list]})
         res = Peer.PeerSet()
-        res.add(Peer.IpBlock(block['cidr'], block.get('except')))
+        try:
+            res.add(Peer.IpBlock(block['cidr'], block.get('except')))
+        except ValueError as e:
+            self.syntax_error(e.args)
```

Comment:
```suggestion
            self.syntax_error(e.args, block)
```
This should provide a context for the error (file and line number)